src/Routes.jsx

```diff
@@ -1,35 +0,0 @@
-import { Route, Switch } from 'react-router-dom';
-import React, { useState } from 'react';
-
-import Login from './pages/signIn';
-import Register from './pages/signUp';
-import HomePage from './pages/home';
-import NewExpense from './pages/newExpense';
-import NewIncome from './pages/newIncome';
-import UserContext from './contexts/UserContext';
-
-export default function Routes() {
-  const [user, setUser] = useState('');
```

Comment:
`user` é uma string? Se não for recomendo inicializar o estado com um objeto vazio